gatsby-config.js

```diff
@@ -28,11 +28,30 @@ module.exports = {
         name: 'images',
       },
     },
+    {
+      resolve: 'gatsby-source-strapi',
+      options: {
+        //apiURL: 'http://localhost:1337',
+	apiURL: 'http://35.197.99.21:1337',
```

Comment:
I set up a DNS record for 35.197.99.21. Let's use `techconnect-api.ddns.net` instead of the raw IP address.

Reply:
Changed and committed